tests/omni_lock_rust/tests/test_opentx.rs

```diff
@@ -626,3 +671,62 @@ fn test_opentx_item_missing() {
     let verify_result = run_opentx_case(config);
     check_res_val(verify_result, vec![1]);
 }
```

Comment:
Add test cases when any of len(input, output, group input, group output) is zero.